releases/development/master/extra/barclamp_install.rb

```diff
@@ -109,15 +109,25 @@ def rm_tmpdir(tmpdir)
     puts "Barclamp at #{bc} has no name, skipping"
     next
   end
+  version = barclamp["barclamp"]["version"].to_i rescue 0
   order = 9999
   if barclamp["crowbar"] and barclamp["crowbar"]["order"] and \
     barclamp["crowbar"]["order"].to_i 
     order = barclamp["crowbar"]["order"].to_i
   end
-  barclamps[name] = { :src => bc, :name => name, :order => order, :yaml => barclamp }
+  barclamps[name] = { :src => bc, :name => name, :order => order, :yaml => barclamp, :version => version }
   debug "barclamp[#{name}] = #{barclamps[name].pretty_inspect}"
 end
 
+debug "checking barclamp versions:"
+barclamps.values.sort_by{|v| v[:order]}.each do |bc|
+  if bc[:yaml]["nav"] && bc[:version] < 1
+    warn "Refusing to install #{bc[:name]} barclamp version < 1 due to incompatible navigation."
```

Comment:
Where is ```warn``` defined? As you can see on my comment for dirk we only implement ```debug``` and ```fatal```.